ynput/ayon-core
# Substance Painter: Allow users to customize the template settings for project creation

## Changelog Description
This PR is to add the support of template settings in the mesh loaders for Substance project creation. User can customize and add template settings in AYON settings and apply it through the option mode(the button with memo icon).

## Additional info
This PR also refactors the original loader's setting without directly import qargparse.
Prefers user to use options (memo icon) to set up their preferences as the prompt dialog might not set up the project as expected(It is still experimental).

Make sure to update your Substance Painter Addon to `0.1.1`

## Testing notes:
1. Customize the template with a template name in AYON setting
![image](https://github.com/ynput/ayon-core/assets/64118225/3a2d8f7d-586f-44a8-b842-16464780dfbf)
2. Launch Substance Painter
3. Click Load..
4. It would pop up the dialog to allow you to choose your template settings
![image](https://github.com/ynput/ayon-core/assets/64118225/5670a580-79e5-47bb-bbd1-4a602e8a8289)
5. Click Ok
6. It should apply all the settings you set in AYON settings
